mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -65,6 +67,77 @@ class Completions(
     (values, result)
   end completions
 
+  private def toCompletionValues(
+      completion: Completion
+  ): List[CompletionValue.Compiler] =
+    val shouldAddSnippet = path match
+      // if the parent is an apply then we already have ()
+      case (fun) :: (appl: GenericApply) :: _ if appl.fun == fun =>
+        false
+      case (_: Import) :: _ => false
+      case _ :: (_: Import) :: _ => false
+      case _ => true
+
+    completion.symbols.flatMap(
+      completionsWithSuffix(_, completion.label, shouldAddSnippet)
+    )
+  end toCompletionValues
+
+  inline private def undoBacktick(label: String): String =
+    label.stripPrefix("`").stripSuffix("`")
+
+  private def findApplySymbols(denot: Denotation): List[Symbol] =
+    denot match
+      case MultiDenotation(denot1, denot2) =>
+        List(findApplySymbols(denot1), findApplySymbols(denot2)).flatten
+      case NoDenotation => Nil
+      case _ =>
+        List(denot.symbol)
+
+  private def findSuffix(
+      shouldAddSnippet: Boolean,
+      methodSymbol: Symbol
+  ): Option[String] =
+    if shouldAddSnippet &&
+      config.isCompletionSnippetsEnabled &&
+      methodSymbol.is(Flags.Method)
+    then
+      val paramss = methodSymbol.paramSymss
+      paramss match
+        case Nil => None
+        case List(Nil) => Some("()")
+        case _ =>
+          val onlyImplicitOrTypeParams = paramss.forall(
+            _.exists { sym =>
+              sym.isType || sym.is(Implicit) || sym.is(Given)
+            }
+          )
+          if onlyImplicitOrTypeParams then None
+          else Some("($0)")
+    else None
+
+  private def completionsWithSuffix(
+      sym: Symbol,
+      label: String,
+      shouldAddSnippet: Boolean
+  ) =
+    // find the apply completion that would need a snippet
+    val methodSymbols = if shouldAddSnippet && sym.is(Flags.Module) then
+      val appl = sym.info.member(nme.apply)
```

Comment:
It's needed to refactor a part with symbolSearch completions a bit to have apply completions for both compiler and workspace found symbols.
Module symbols might come from `enrichWithSymbolSearch` which is called in `filterInteresting` while here we do these checks only for symbols returned by compiler.

Reply:
Do you have an example in mind maybe?

Reply:
Ok, I found an example and fixed!